packages/boot/src/types.ts

```diff
@@ -30,7 +30,7 @@ export type ArtifactOptions = {
    * A `glob` string to use when searching for files. This takes
    * precedence over other options.
    */
-  glob?: string;
+  globs?: string | string[];
```

Comment:
This looks like a breaking change to me, can you please confirm?

Should we preserve backwards compatibility e.g. by computing the final list of glob patterns by concatenaning `globs` and `glob`?